src/self-model.scm

```diff
@@ -660,12 +662,12 @@
 
 ;; Glance at the last known location of a face that is no longer
 ;; visible.
-(DefineLink
-	(DefinedSchema "glance at lost face")
-	(Put
-		(Evaluation (GroundedPredicateNode "py:glance_at_face")
-			(ListLink (Variable "$face-id")))
-		(DefinedSchema "New departures")))
+;(DefineLink
+;	(DefinedSchema "glance at lost face")
+;	(Put
+;		(Evaluation (GroundedPredicateNode "py:glance_at_face")
+;			(ListLink (Variable "$face-id")))
+;		(DefinedSchema "New departures")))
 
```

Comment:
You have an absolutely terrible habit of commenting out code, instead of removing it.  You have to break this habit.  If your changes are correct, then please just remove the code. If you are not sure if your changes are correct, then you should not be committing the code.   There are not many plausible, reasonable situation where its correct to comment out code.  This doesn't seem to be one of them.